frontends/web/src/routes/account/account.tsx

```diff
@@ -417,5 +419,5 @@ const loadHOC = load<LoadedAccountProps, AccountProps & TranslateProps>(({ code
     config: 'config',
 }))(Account);
 
-const HOC = translate<AccountProps>()(loadHOC);
+const HOC = translate()(loadHOC);
 export { HOC as Account };
```

Comment:
Test buttons on account overview

Reply:
Bug: for some reason I the account-info link is gone 🤔

Reply:
Do you have more info on this bug so i can check it? (how to reproduce it, expected result)

Reply:
Yes sorry, on account overview there should be a account info link in the header, with useful information such as xpub or coin type.

The link is passed as child to the header component, for some reason this doesn't render anymore.

Also probably this shouldn't be a `<a href>` I think, we should always route programatically.

https://github.com/digitalbitbox/bitbox-wallet-app/blob/ad414e3dad2e8926c90660fad8bb26a16affa772/frontends/web/src/routes/account/account.tsx#L307-L310

But that it doesn't render at all is maybe due to `hasChildren` here https://github.com/digitalbitbox/bitbox-wallet-app/blob/ad414e3dad2e8926c90660fad8bb26a16affa772/frontends/web/src/components/layout/header.tsx#L42